pilot/pkg/networking/plugin/authz/rbac_v2_test.go

```diff
@@ -16,16 +16,6 @@ import (
 
 func TestConvertRbacRulesToFilterConfigV2(t *testing.T) {
 	roles := []model.Config{
-		{
```

Comment:
Can we create new test case instead of removing existing one?

Reply:
I put `service-role-1` back and used it in one of the tests. I replaced `roleRef service-role-1` by `actions`. This should cover same cases as before. As far as `rbac v2` is concerned, I think we only need to focus on testing `AuthorizationPolicy` CRD, since `ServiceRole` has been tested extensively in `rbac_test.go`. 
One thing I'd like to mention is that writing unit tests for `rbac v2` is very tricky and tedious, some of it because we allow duplicated principals/permissions, so we'll have a really long map for each rbac conversion. We can think about how we can refactor this, maybe we can follow your approach on `rbac v1` unit tests once the PR has been checked in.